doc/rados/operations/erasure-code-clay.rst

```diff
@@ -228,7 +228,7 @@ can recover from the failure of any ``m`` OSDs.
 	+=================+================+=================+==================================+
 	| (k=10, m=4)     | 7 * S, 0.6 (d=7)                 | 3.25 * S, 0.4 (d=13)             |
 	+-----------------+----------------------------------+----------------------------------+
-	| (k=16, m=4)     | 4 * S, 0.5 (d=5)                 | 4.75 * S, 0.25 (d=19)            |
+	| (k=16, m=4)     | 4 * S, 0.5625 (d=4)              | 4.75 * S, 0.25 (d=19)            |
```

Comment:
the change looks good. but 

could you prefix the title of your commit message with the subcomponent your are changing ? see https://github.com/ceph/ceph/blob/master/SubmittingPatches.rst#3-describe-your-changes, in this case, it would be "doc/rados:".

could you add a "`Signed-off-by`" line at the end of your commit message? "git commit -s" will do the trick for you. see https://github.com/ceph/ceph/blob/master/SubmittingPatches.rst#1-sign-your-work